src/components/AdminAccess.vue

```diff
@@ -6,6 +6,9 @@
 <template>
 	<div class="admin">
 		<h3>{{ t('privacy', 'Administrators') }}</h3>
+		<p class="admin__desc">
+			{{ t('privacy', 'The following users are designated administrators and may potentially have access to your data through system storage or backups. However, they do not have easy access to your data via the web interface or client applications.') }}
```

Comment:
```suggestion
			{{ t('privacy', 'The following accounts are designated administrators and may potentially have access to your data through system storage or backups. However, they do not have easy access to your data via the web interface or client applications.') }}
```

Only super tiny wording detail. :) Otherwise a really good improvement!